Build/Build.cs

```diff
@@ -69,13 +69,20 @@ class Build : NukeBuild
 
     string SemVer;
 
-    Target Clean => _ => _
-        .OnlyWhenDynamic(() => RunAllTargets || HasSourceChanges)
+    Target Informations => _ => _
```

Comment:
🔧 That's a normal English word...

Reply:
Removed the target, since I only used it for debugging :)